frameworks/cassandra/docs/managing.md

```diff
@@ -74,6 +74,30 @@ For example, let's say `node-2`'s host system has died and `node-2` needs to be
 dcos beta-cassandra pod replace node-2
 ```
 
+## Seed nodes
+Cassandra seed nodes are those nodes with indices smaller than the seed node count.  By default Cassandra is deployed 
+with a seed node count of two.  So `node-0` and `node-1` are seed nodes.  When a replace opration is performed on one 
+these nodes all other nodes must be restarted to be brought up to date regarding the ip address of the new seed node.
+This is performed automatically.  For example if `node-0` needed to be replaced we would execute:
```

Comment:
Suggest, "So, `node-0` and `node-1` are seed nodes.  When a replace operation is performed on one 
 these nodes, all other nodes must be restarted to be brought up to date regarding the ip address of the new seed node. This is operation is performed automatically."